ces_client/utils.py

```diff
@@ -65,4 +68,20 @@ def make_answers(self, state):
         answers.append({'text': 'Goodbye!',
                         'value': settings.DECISIONTREE_SESSION_END_TRIGGER})
 
-        return answers
\ No newline at end of file
+        return answers
+
+    @classmethod
+    def prep_text_for_botui(self, text):
```

Comment:
For class methods, the first argument is not an instance but the class object. The convention, is for the name of the first argument to be `cls`

Reply:
oh, yes